nixos/tests/ft2-clone.nix

```diff
@@ -0,0 +1,35 @@
+import ./make-test-python.nix ({ pkgs, ... }: {
+  name = "ft2-clone";
+  meta = with pkgs.lib.maintainers; {
```

Comment:
One small remark: I think it would be better to limit the `with` expression to a smaller scope, i.e. only to the `maintainers` entry.

Reply:
Yes, honestly I'm not sure why I did it this way